packages/block-editor/src/components/list-view/block-contents.js

```diff
@@ -32,17 +32,21 @@ const ListViewBlockContents = forwardRef(
 	) => {
 		const { clientId } = block;
 
-		const { blockMovingClientId, selectedBlockInBlockEditor } = useSelect(
+		const {
+			blockMovingClientId,
+			selectedBlockInBlockEditor,
+			selectedBlocks,
+		} = useSelect(
 			( select ) => {
 				const {
-					getBlockRootClientId,
 					hasBlockMovingClientId,
 					getSelectedBlockClientId,
+					getSelectedBlockClientIds,
 				} = select( blockEditorStore );
 				return {
-					rootClientId: getBlockRootClientId( clientId ) || '',
```

Comment:
As of https://github.com/WordPress/gutenberg/pull/33320 the `rootClientId` is no longer used in this function, so I think it can safely be removed here.